MAUI/CheckBox/Visual-Customization.md

```diff
@@ -0,0 +1,195 @@
+    stackLayout.Children.Add(intermediate);
+    this.Content = stackLayout;
+
+{% endhighlight %}
+{% endtabs %}
+
+![.NET MAUI CheckBox](Images/Visual-Customization/statecolor.png)
+
+## StrokeThickness
+
+The tick box stroke thickness of the checkbox control can be customized using the [`StrokeThickness`] property.  
+
+{% tabs %}
+{% highlight xaml %}
+
+    <StackLayout>
+        <syncfusion:SfCheckBox Text="Hello" StrokeThickness="2" UncheckedColor="Blue" FontSize="20"/>
+        <syncfusion:SfCheckBox Text="Hello" StrokeThickness="4" UncheckedColor="Blue" FontSize="25"/>
+        <syncfusion:SfCheckBox Text="Hello" StrokeThickness="6" UncheckedColor="Blue" FontSize="30"/>
+    </StackLayout>
+
+{% endhighlight %}
+{% highlight c# %}
+
+    StackLayout stackLayout = new StackLayout();
+    SfCheckBox check1 = new SfCheckBox();
+    check1.Text = "Hello";
+    check1.StrokeThickness = 2;
+    check1.FontSize = 20;
+    check1.UncheckedColor = Colors.Blue;
+    SfCheckBox check2 = new SfCheckBox();
+    check2.Text = "Hello";
+    check2.StrokeThickness = 4;
+    check2.FontSize = 25;
+    check2.UncheckedColor = Colors.Blue;
+    SfCheckBox check3 = new SfCheckBox();
+    check3.Text = "Hello";
+    check3.StrokeThickness = 6;
+    check3.FontSize = 30;
+    check3.UncheckedColor = Colors.Blue;
+    stackLayout.Children.Add(check1);
+    stackLayout.Children.Add(check2);
+    stackLayout.Children.Add(check3);
+    this.Content = stackLayout;
+
+{% endhighlight %}
+{% endtabs %}
+
+![.NET MAUI CheckBox](Images/Visual-Customization/strokethickness.png)
+
+## Setting a caption text appearance
+
+You can customize the display text appearance of the [`SfCheckBox`] control using the following properties:
+
+* [`TextColor`]: Changes the color of the text.
+* [`HorizontalTextAlignment`]: Changes the horizontal alignment of the caption text.
+* [`FontFamily`]: Changes the font family of the caption text.
+* [`FontAttributes`]: Sets font attributes(bold/italic/none) of the caption text.
+* [`FontSize`]: Sets font size of the caption text.
+
+{% tabs %}
+{% highlight xaml %}
+
+    <syncfusion:SfCheckBox x:Name="caption" Text="CheckBox" IsChecked="True" TextColor="Violet" HorizontalTextAlignment="Center" FontFamily="Arial" FontAttributes="Bold" FontSize="20"/>           
+
+{% endhighlight %}
+{% highlight c# %}
+
+    SfCheckBox caption = new SfCheckBox();
+    caption.IsChecked = true;
+    caption.Text = "CheckBox";
+    caption.TextColor = Color.Violet;
+    caption.HorizontalTextAlignment = TextAlignment.Center;
+    caption.FontFamily = "Arial";
+    caption.FontAttributes = FontAttributes.Bold;
+    caption.FontSize = 20;
+    this.Content = caption;
+
+{% endhighlight %}
+{% endtabs %}
+
+![.NET MAUI CheckBox](Images/Visual-Customization/textappearance.png)
+ 
+## TickColor Customization
+
+The [`TickColor`] property customizes the color of the tick in [`SfCheckBox`] control using the following properties:
```

Comment:
The [`TickColor`] property customizes the color of the tick in the [`SfCheckBox`] control using the properties listed below.